spectractor/extractor/spectroscopy.py

```diff
@@ -526,7 +519,7 @@ def print_detected_lines(self, output_file_name="", overwrite=False, print_table
 # O2 = Line(762.2, atmospheric=True, label=r'$O_2$',  # 762.2 is a weighted average of the O2 line simulated by Libradtran
 #           label_pos=[0.007, 0.02],
 #           use_for_calibration=True)  # http://onlinelibrary.wiley.com/doi/10.1029/98JD02799/pdf
-O2_1 = Line(760.3, atmospheric=True, label='',
+O2_1 = Line(760.3, atmospheric=True, label='$O_2$',
```

Comment:
It's great that this label is now added. I wonder how hard it would be to make the label only appear once the plots so that it doesn't look weird (because they're so close to each other). I wouldn't worry if that's hard or might cause bugs though, but just a thought.

Reply:
OK, I fix it with `label_pos`